rb/lib/selenium/webdriver/common/manager.rb

```diff
@@ -47,7 +47,7 @@ def add_cookie(opts = {})
 
```

Comment:
Can you update the documentation of the `add_cookie` method to include a new parameter and mention it's Chrome-only?

Reply:
Updated as `  # @option opts [String] :sameSite (Strict or Lax) currently supported only in chrome 80+ versions`